### PR TITLE
Update SUPPORT.md

### PR DESCRIPTION
**Pull request by thambu1710**

### CODE REVIEW

**Comment by TerWebz** [comment]
Looks good to me!

**Comment by thundron** [comment]
Sorry for the delay in picking this up! 🙏 
Since this is only a documentation change you should add a `Change-type: none` footer tag to your commit.
That will make the CI skip all the package building so we can merge without creating a new version.
Other than that, looks good to me!

**Comment by shawaj** [comment]
@zvin @thundron is this something we actually want to do? Update from `Etcher` to `balenaEtcher`?

File: SUPPORT.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-Getting help with Etcher
```
Comment (karakartal1881) [inline SUPPORT.md:1]: https://github.com/balena-io/etcher/pull/3209/commits/e414f15c7e46c70156e983f856e33158fb174fbb#diff-c5fe610b0215b144474106251cab954f8af55fe26cbd5d2265fd6ca19109c97b

**Review by karakartal1881 — APPROVED** [review]

**Comment by thundron** [comment]
cherry-picked in https://github.com/balena-io/etcher/pull/3562 , waiting for some internal testing to merge 👍

**Pull request closed without merging**